docs/csharp/fundamentals/coding-style/coding-conventions.md

```diff
@@ -1,332 +1,96 @@
-
-- Write only one statement per line.
-- Write only one declaration per line.
-- If continuation lines are not indented automatically, indent them one tab stop (four spaces).
-- Add at least one blank line between method definitions and property definitions.
-- Use parentheses to make clauses in an expression apparent, as shown in the following code.
-
-  :::code language="csharp" source="./snippets/coding-conventions/program.cs" id="Snippet2":::
-  
-## Place the using directives outside the namespace declaration
-
-When a `using` directive is outside a namespace declaration, that imported namespace is its fully qualified name. That's more clear. When the `using` directive is inside the namespace, it could be either relative to that namespace or it's fully qualified name. That's ambiguous.
-
-```csharp
-using Azure;
-
-namespace CoolStuff.AwesomeFeature
-{
-    public class Awesome
-    {
-        public void Stuff()
-        {
-            WaitUntil wait = WaitUntil.Completed;
-            …
-        }
-    }
-}
-```
-
-Assuming there is a reference (direct, or indirect) to the <xref:Azure.WaitUntil> class.
-
-Now, let's change it slightly:
-
-```csharp
-namespace CoolStuff.AwesomeFeature
-{
-    using Azure;
-    
-    public class Awesome
-    {
-        public void Stuff()
-        {
-            WaitUntil wait = WaitUntil.Completed;
-            …
-        }
-    }
-}
-```
-
-And it compiles today. And tomorrow. But then sometime next week this (untouched) code fails with two errors:
-
-```console
-- error CS0246: The type or namespace name 'WaitUntil' could not be found (are you missing a using directive or an assembly reference?)
-- error CS0103: The name 'WaitUntil' does not exist in the current context
-```
-
-One of the dependencies has introduced this class in a namespace then ends with `.Azure`:
-
-```csharp
-namespace CoolStuff.Azure
-{
-    public class SecretsManagement
-    {
-        public string FetchFromKeyVault(string vaultId, string secretId) { return null; }
-    }
-}
-```
-
-A `using` directive placed inside a namespace is context-sensitive and complicates name resolution. In this example, it's the first namespace that it finds.
+> These guidelines are used by Microsoft to develop samples and documentation. They were adopted from the [.NET Runtime, C# Coding Style](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md) and [C# compiler (roslyn)](https://github.com/dotnet/roslyn/blob/main/CONTRIBUTING.md#csharp) guidelines. We chose those guidelines because they have been tested over several years of Open Source development. They've helped community members participate in the runtime and compiler projects. They are meant to be an example of common C# conventions, and not an authoritative list (see [Framework Design Guidelines](../../../standard/design-guidelines/index.md) for that).
```

Comment:
```suggestion
> These guidelines are used by Microsoft to develop samples and documentation. They were adopted from the [.NET Runtime, C# Coding Style](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md) guidelines. We chose those guidelines because they have been tested over several years of Open Source development. They've helped community members participate in the runtime and compiler projects. They are meant to be an example of common C# conventions, and not an authoritative list (see [Framework Design Guidelines](../../../standard/design-guidelines/index.md) for that).
```

Nit: I removed the C# compiler (roslyn) reference here, since I do not believe we're using theirs.